pkgs/development/python-modules/teslajsonpy/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, aiohttp
+, backoff
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, wrapt
+}:
+
+buildPythonPackage rec {
+  pname = "teslajsonpy";
+  version = "0.10.4";
+
+  src = fetchFromGitHub {
+    owner = "zabuldon";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "18frynmy47i9c24mdy819y2dnjwmhnmkly5mbmhikpbmm6d0yjf1";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    backoff
+    wrapt
+  ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  # Not all Home Assistant related check pass
+  disabledTests = [ "test_values_on_init" ];
```

Comment:
Looks like an issue with the mock data. Upstream doesn't seem to run a CI, so hard to tell if it is working for them.

Reply:
https://github.com/zabuldon/teslajsonpy/issues/121

Reply:
It's not in our CI but I just checked and it passes locally. It's also a simple dictionary lookup with mock data so I'm not sure how it could fail.  Is there a quick way to set up a NixOS test environment?